data/entri.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: ENTRI
+description: Vernacular EdTech Platform
+founded: May 2017
+based: Product based
+domain: EdTech
+logo: entri.png
+location: Kochi
+address: Maleppally Rd, Thrikkakara, Kakkanad, Kerala 682021
+technologies: Django, Kotlin, React
```

Comment:
✨Thanks for taking your time to contribute. Please refer the existing data on how to represent `technologies` as I found a small problem with your mdx code

Reply:
Thanks for the clarification. This change has been made.